addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting1Message.java

```diff
@@ -110,6 +112,8 @@ public static Commands fromByte(int input) throws RFXComUnsupportedValueExceptio
     public byte signalLevel;
     public boolean group;
 
+    private static byte[] lastUnit = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
I personally don't like this kind of state do other parts of the rfxcom binding already use such techniques. Maybe others have more or other ideas about this?

And is it correct that is never reset?